src/diffusers/pipelines/animatediff/pipeline_animatediff.py

```diff
@@ -389,13 +389,23 @@ def prepare_ip_adapter_image_embeds(
                     [single_negative_image_embeds] * num_images_per_prompt, dim=0
                 )
 
-                if self.do_classifier_free_guidance:
+                if do_classifier_free_guidance:
```

Comment:
adding `do_classifier_free_guidance` as an argument to `prepare_ip_adapter_imabe_embeds` so we can use this pipeline method to save image embeddings